packages/functional-tests/playwright.config.ts

```diff
@@ -36,6 +36,14 @@ export default defineConfig<PlaywrightTestConfig<TestOptions, WorkerOptions>>({
   // Retry on CI only.
   retries: CI ? 1 : 0,
 
+  // Total allowable time spent for the test function, fixtures, beforeEach and afterEach hooks. Defaults to 30 seconds.
+  timeout: 60000, // 1 minute
+
+  // Report slow test files
+  // max: The maximum number of slow test files to report. Defaults to 5.
+  // threshold: Test duration in milliseconds that is considered slow. Defaults to 15 seconds.
+  reportSlowTests: { max: 5, threshold: 30000 }, // 30 seconds
```

Comment:
@pdehaan added your suggestion to include [reportSlowTests](https://playwright.dev/docs/api/class-testconfig#test-config-report-slow-tests)

Reply:
:+1: I looked through the 8 shards' logs and didn't see any obvious output of slow tests, but lets roll with it! ❤️

Reply:
I'm not sure if it's working actually, looking at the latest nightly, there are files over the 30 seconds.
syncV3/fxDesktopHandshakeNonSync.spec.ts, 5, 30.294
Test File | Cases # | Time (s)
--- | --- | ---
oauth/oauthPermissionsSignin.spec.ts | 5 | 30.866
oauth/oauthPromptNone.spec.ts | 8 | 31.191
key-stretching-v2/totp.spec.ts | 2 | 31.426
subscription-tests/support.spec.ts | 3 | 32.007
subscription-tests/coupons-ui-tests/couponForeverDiscount.spec.ts | 2 | 32.847
signin/signinBlocked.spec.ts | 5 | 36.733
misc.spec.ts | 3 | 39.115
key-stretching-v2/signup.spec.ts | 4 | 39.143
syncV3/signIn.spec.ts | 7 | 42.815
subscription-tests/coupons-ui-tests/ui-tests.spec.ts | 1 | 45.112
key-stretching-v2/signinBlocked.spec.ts | 4 | 47.812
settings/changeEmail.spec.ts | 5 | 51.302
signin/signinCached.spec.ts | 6 |51.422
react-conversion/oauthSignin.spec.ts | 6 | 71.306
settings/totp.spec.ts | 7 | 80.664
key-stretching-v2/changePassword.spec.ts | 8 | 88.641
subscription-tests/subscription.spec.ts | 8 | 90.109
react-conversion/signInRelyingParties.spec.ts | 2 | 108.699